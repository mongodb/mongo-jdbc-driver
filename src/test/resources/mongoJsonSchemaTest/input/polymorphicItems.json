{"bsonType": "object",
  "properties": {
    "bar": {
      "bsonType": "object",
      "properties": {
        "a": {
          "bsonType": "array",
          "items": {"bsonType":  "date"}
        },
        "b": {
          "bsonType": "array",
          "items": [
            {
              "anyOf": [
                {
                  "bsonType": "string"
                },
                {
                  "bsonType": "null"
                }
              ]
            },
            {
              "bsonType": "boolean"
            }
          ],
          "additionalProperties": false
        },
        "c": {"bsonType": "objectId"}
      }
    }
  }
}
