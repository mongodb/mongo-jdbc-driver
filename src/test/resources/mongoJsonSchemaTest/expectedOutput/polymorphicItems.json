{
  "bsonType":"object",
  "properties":{
    "bar":{
      "bsonType":"object",
      "properties":{
        "a":{
          "bsonType":"array",
          "items":{}
        },
        "b":{
          "bsonType":"array",
          "items":{},
          "additionalProperties":false
        },
        "c":{
          "bsonType":"objectId"
        }
      }
    }
  }
}
